docs/source/installation/development.rst

```diff
@@ -3,5 +3,123 @@
 Installation guide (development)
 ===============================
 
-.. todo::
-    Write guide
+Install System Packages
+-----------------------
+
+We'll assume you've installed PostgreSQL (and its ``contrib package``, if on a Linux system) and managed to setup/start
+the server. PostgreSQL 9.6 at least is required. You can find instructions on how to install it on Fedora/CentOS
+`here <https://www.if-not-true-then-false.com/2012/install-postgresql-on-fedora-centos-red-hat-rhel/>`_.
+You can find the same for Debian/Ubuntu `here <https://www.tecmint.com/install-postgresql-on-debian-and-ubuntu/>`_.
```

Comment:
I'd explain the steps here (almost copy&paste from the prod setup guides) instead of linking to external blogs that may or may not break in the future. Or at least link to something more official, like https://wiki.postgresql.org/wiki/Apt and https://wiki.postgresql.org/wiki/YUM_Installation